coverage.sh

```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env nash
```

Comment:
kkk I renamed the script to cover.sh because I'm lazy kkk If you dislike we can use coverage.sh :-)